composer.json

```diff
@@ -25,6 +25,7 @@
         "nyholm/psr7": "^1.3",
         "psr/http-client": "^1.0",
         "symfony/process": "^4.4.30 | ^5.0 |^6.0 | ^7.0",
+        "symfony/cache": "^4.4.30 | ^5.0 |^6.0 | ^7.0",
```

Comment:
In practice, caching is optional so I would require the PSR packages instead here that don't provide any implementations